lib/sandbox.macos.ml

```diff
@@ -54,12 +54,20 @@ let copy_to_log ~src ~dst =
    For macOS we also need to create the illusion of building in a static
    home directory, and to achieve this we copy in the pre-build environment
    and copy back out the result. It's not super efficienct, but is necessary.*)
-let post_build ~result_dir ~home_dir (t : t) =
-  Os.sudo ["rsync"; "-aHq"; "--delete"; home_dir ^ "/"; result_dir ] >>= fun () ->
+
+let unmount_fuse (t : t) =
```

Comment:
```suggestion
let unmount_fuse t =
```
The compiler will infer the right type for this, don't worry about the type annotation.
Same for `post_build` and `post_cancellation`.

Reply:
Removed all occurrences of `(t : t)` except for the `run` function where it seems to be needed.